params/version.go

```diff
@@ -55,10 +55,13 @@ func ArchiveVersion(gitCommit string) string {
 	return vsn
 }
 
-func VersionWithCommit(gitCommit string) string {
+func VersionWithCommit(gitCommit, gitDate string) string {
 	vsn := VersionWithMeta
 	if len(gitCommit) >= 8 {
 		vsn += "-" + gitCommit[:8]
 	}
+	if VersionMeta != "stable" {
+		vsn += "-" + gitDate
```

Comment:
Question regarding this. We do want the date anytime it's not stable, so by nature a `v1.9.0-unstable-` would be an error. The question is, do we want to drop the check here (essentially a bandaid) and drop the extra dash, or no? I think we do because the other apps built, ie Swarm, end up with this extra dash because this process isn't fully built out there.

Reply:
fixed